packages/devtools_app/lib/src/provider/provider_screen.dart

```diff
@@ -0,0 +1,805 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:pedantic/pedantic.dart';
+
+import '../common_widgets.dart';
+import '../config_specific/logger/logger.dart';
+import '../eval_on_dart_library.dart';
+import '../globals.dart';
+import '../inspector/inspector_text_styles.dart';
+import '../inspector/inspector_tree.dart';
+import '../screen.dart';
+import '../split.dart';
+import 'eval.dart';
+import 'provider_state_controller.dart';
+
+const kTypeColor = Color.fromARGB(255, 78, 201, 176);
+const kBoolColor = Color.fromARGB(255, 86, 156, 214);
+const kNullColor = kBoolColor;
+const kNumColor = Color.fromARGB(255, 181, 206, 168);
+const kStringColor = Color.fromARGB(255, 206, 145, 120);
+const kPropertyColor = Color.fromARGB(255, 206, 145, 120);
+
+const double rowHeight = 20.0;
+const double horizontalSpacing = 15;
+const double expandIconSize = 20;
```

Comment:
would be nice to keep the icons a consistent size with the other icons in devtools. there should be an existing const for this.